src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final int firstSide;
+    private final int secondSide;
+
+    public Rectangle(String color, int firstSide, int secondSide) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    @Override
+    public double getArea() {
+        return firstSide * secondSide;
+    }
+
+    @Override
+    public void toDrawFigure() {
+        System.out.println("Figure: rectangle, area: " + getArea() + " sq. units, "
+                + "first side: " + firstSide + " units, second side: " + secondSide
+                + " units, color: " + color);
```

Comment:
Use the `name()` method for getting String representation of enum constants instead of `toString()` as per the checklist. If `color` is an enum, you should call `color.name()`.